portfolio.css

```diff
@@ -0,0 +1,1512 @@
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0;
+}
+
+.section4 {
+  background: #fff;
+  width: 375px;
+}
+
+.latsSection {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+}
+
+.always {
+  position: relative;
+  width: 327px;
+  height: 216px;
+  left: 0;
+  top: 0;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 24px;
+}
+
+.forms {
+  margin-top: 40px;
+}
+
+.input {
+  position: relative;
+  height: 44px;
+  left: 16px;
+  right: 16px;
+  top: calc(50% - 18px / 2);
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #979493;
+  margin-top: 12px;
+  margin-bottom: -12px;
+  margin-left: 24px;
+  border: none;
+}
+
+.input:hover {
+  text-decoration: underline;
+  text-decoration-color: #0b5be6;
+  border: none;
+}
+
+.text-area {
+  position: relative;
+  left: 12px;
+  right: 12px;
+  top: 12px;
+  bottom: 12px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  margin-left: 24px;
+  color: #3c3a39;
+  background: #f2f2f2;
+  border: none;
+}
+
+.text-area:hover {
+  border: 2px solid #0b5be6;
+  text-decoration: underline;
+  text-decoration-color: #0b5be6;
+}
+
+.touch > button {
+  position: relative;
+  width: 150px;
+  height: 44px;
+  left: 12px;
+  top: 12px;
+  margin-top: 44px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 12px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  background: #ff6b00;
+  margin-bottom: 24px;
+  margin-left: 80px;
+  border: none;
+}
+
+.touch > button:hover {
+  border: 2px solid #0b5be6;
+  text-decoration: underline;
+  text-decoration-color: #0b5be6;
+}
+
+.images-last:hover {
+  filter: brightness(20%);
+  filter: drop-shadow(34%);
+}
+
+.social-media1 > ul {
+  display: flex;
+}
+
+.social-1 {
+  position: relative;
+  height: 16px;
+  padding: 16px;
+  filter: grayscale(200%);
+  filter: contrast(80%);
+}
+
+.images1 {
+  font-size: 30px;
+  filter:
+    invert(38%) sepia(41%) saturate(291%) hue-rotate(179deg)
+    brightness(88%) contrast(97%);
+}
+
+.shape > img {
+  margin-left: 100px;
+  margin-bottom: 24px;
+  align-items: center;
+}
+
+.section2p {
+  width: 375px;
+  display: none;
+  position: relative;
+  background: rgba(0, 0, 0, 0.5);
+  overflow-y: hidden;
+  backdrop-filter: blur(10px);
+  overflow-x: hidden;
+}
+
+.detail {
+  width: 343px;
+  background: #fff;
+  padding-left: 20px;
+  padding-right: 20px;
+}
```

Comment:
- `required`
 
- [ ] Amazing job so far!! :clap: :clap: 

- [ ] Kindly make the `pop-up cards` as similar as possible to the figma template.

- [ ] Please see to it that if someone clicks on see project, it pops up right there, and not else where

- [ ]  also see to it the `fonts` `margins` and `responsivements` is working fine

 

- [ ] please see screenshots below


- ` your template`
![image](https://user-images.githubusercontent.com/66230306/180577947-97eb6f5b-13b0-480a-af31-c68faeb855d0.png)



- `figma template`
![image](https://user-images.githubusercontent.com/66230306/180577789-d75fa9e1-9847-47ec-ba34-2ddbd54590f4.png)